pymemcache/test/test_client_hash.py

```diff
@@ -42,7 +43,7 @@ def test_setup_client_without_pooling(self):
         client_class = 'pymemcache.client.hash.HashClient.client_class'
         with mock.patch(client_class) as internal_client:
             client = HashClient([], timeout=999, key_prefix='foo_bar_baz')
-            client.add_server('127.0.0.1', '11211')
+            client.add_server(('127.0.0.1', '11211'))
```

Comment:
It doesn't look like we have any tests that use the `.add_server(server, port=port)` signature now?

Reply:
https://github.com/pinterest/pymemcache/pull/295/files#diff-dfcf37102448920749549449cc837e8aR382-R399

Reply:
Perfect. Thanks for the link!